server/middleware/auth.js

```diff
@@ -1,20 +1,23 @@
 const jwt = require("jsonwebtoken");
 const keys = require("../config/keys");
 
+//  TODO: Refactor cookie name "auth-token" into
+//  a variable? (also used in user/authenticate route)
 module.exports = function (req, res, next) {
-  //get the token from the header
-  const token = req.header("x-auth-token");
+  // Get token from http-only cookie
```

Comment:
To make this consistent with validator, perhaps it should be named authenticator rather than auth?